frontend/javascripts/oxalis/workers/slow_byte_array_lz4_compression.worker.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+// NOTE: This is a mirror of byte_array_lz4_compression.worker.js
+// and is ONLY meant for mocking during tests. This implementation
+// allows to introduce an artificial delay for compression/decompression.
+
+import lz4 from "lz4js";
+
+import { sleep } from "libs/utils";
+
+import { expose } from "./comlink_wrapper";
+
+let isSleepEnabled = false;
+
+export function enableSlowCompression(isEnabled: boolean) {
+  isSleepEnabled = isEnabled;
+}
+
+async function slowCompressLz4Block(data: Uint8Array, compress: boolean): Promise<Uint8Array> {
+  if (isSleepEnabled) {
+    await sleep(400);
+  }
+  if (compress) {
```

Comment:
Would it work to export `compressLz4Block` in the original file to import it here and use it directly? This would get rid of the duplication and safeguard against desynchronization of the two files. Or would you be worried that someone imports `compressLz4Block` and uses it directly in a production setting, accidentally then?

Reply:
> Or would you be worried that someone imports compressLz4Block and uses it directly in a production setting, accidentally then?

Yes, that's what I was worried about. However, I now defined a `__compressLz4BlockHelper` function within the actual compression worker which is then imported. This reduces the duplication of the actual compress calls and the name (plus docstring) should make clear that it should not be imported somewhere else. I hope that's better now!